### DISCUSSION TITLE
[开发] Minecraft 中的绳网客户端？

### DISCUSSION THREAD

**Discussion by MetsukiMio** [share121/inter-knot#79] [其他]
![屏幕截图_20240809_163956](https://github.com/user-attachments/assets/77a6ca5b-874c-4e20-9469-a7a836107d59)
其实这个项目从去年12月就开始开发了，然后就把 Minecraft 的资源包列表改得到处都是圆角（
![屏幕截图_20240809_164210](https://github.com/user-attachments/assets/f3e01e00-522e-4e62-9d0b-887b8f17870f)
以及这个用2200行yaml暴力枚举写出来的贪吃蛇
项目已经开源在https://github.com/PrimogemStudio/Advanced-Framework，也可以当作计算机图形学的范例，这里面还藏了一个 MMD 渲染器，希望在大重构后能写出一个客户端吧（

**Comment by Modog1145** [comment]
牛啊

**Comment by AL-Pinecore** [comment]
牛逼啊

**Comment by 434-t** [comment]
牛逼啊

**Comment by Hexiaoling457** [comment]
牛啊

**Comment by NOVAQUQ** [comment]
看不懂思密达ʕ(ⓛ–ⓛ)ʔ

**Comment by Wattloo** [comment]
6啊

**Comment by soultgp** [comment]
牛蛙！
